lib/SILOptimizer/Transforms/AllocBoxToStack.cpp

```diff
@@ -364,12 +422,13 @@ static InFlightDiagnostic diagnose(ASTContext &Context, SourceLoc loc,
 /// canPromoteAllocBox - Can we promote this alloc_box to an alloc_stack?
 static bool canPromoteAllocBox(AllocBoxInst *ABI,
                                SmallVectorImpl<Operand *> &PromotedOperands) {
+  SmallPtrSet<SILFunction *, 8> VisitedCallees;
   // Scan all of the uses of the address of the box to see if any
   // disqualifies the box from being promoted to the stack.
   if (auto *User = findUnexpectedBoxUse(ABI,
```

Comment:
In a follow on commit, can you add a comment here saying that this initializes PromotedOperands from our recursive dfs walk of the call graph. I just want that to be very clear. Right now you need to read the code to understand that. I shouldn't have to read the code.